CMakeLists.txt

```diff
@@ -96,7 +96,10 @@ set_property( CACHE CMAKE_CUDA_ARCHITECTURES PROPERTY STRINGS
 
 #-----------------------------------
 # BLAS options
-# SLATE defaults to single-threaded BLAS.
+# Auto-detect blas_threaded support (single-threaded, multi-threaded)
+# Using blas_threaded=false causes OneMKL configuration problems
+# because OneMKL requires multi-threaded BLAS.  Note: This setting may
+# need to be revisited if a BLAS library is not OpenMP-aware.
```

Comment:
"Update comment above to match code, or just remove comment altogether."
This clarifies the requirement from OneMKL.  
Previously, the single-threaded blas library was found and incorrectly linked with -lmkl_gf_lp64.

Reply:
ESSL is actually written with OpenMP, and uses OMP_NUM_THREADS. There is no ESSL_NUM_THREADS, as far as I know, unlike MKL_NUM_THREADS. I just don't know if ESSL handles being single-threaded inside a parallel section the same way MKL does, or if we will end up with OMP_NUM_THREADS^2 threads.

Reply:
For ESSL, there are different libraries for serial and multithreaded, you link with the desired version.
https://www.ibm.com/docs/en/essl/6.3?topic=use-serial-smp-libraries-provided-by-essl
In /sw/summit/essl/6.3.0/essl/6.3/lib64
libessl.so - single threaded
libesslsmp.so - smp version

Reply:
The Makefile links with libessl.so, but CMake in the current master links with libesslsmp.so:
```
[mgates31@login2.summit build-summit]$ git idx
b2b08cf3 trsmA: remove the use of transpose for the scaling and setting of tiles (#88). Fixes #66.

[mgates31@login2.summit build-summit]$ cmake ..
-- The CXX compiler identification is GNU 9.1.0
-- The Fortran compiler identification is GNU 9.1.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /sw/summit/gcc/9.1.0-alpha+20190716/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting Fortran compiler ABI info
-- Detecting Fortran compiler ABI info - done
-- Check for working Fortran compiler: /sw/summit/gcc/9.1.0-alpha+20190716/bin/gfortran - skipped
-- Setting CMAKE_INSTALL_PREFIX = /opt/slate
---------------------------------------- C API: c_api = OFF
-- slate_id = b2b08cf3

---------------------------------------- OpenMP: use_openmp = ON
-- Found OpenMP_CXX: -fopenmp (found version "4.5") 
-- Found OpenMP_Fortran: -fopenmp (found version "4.5") 
-- Found OpenMP: TRUE (found version "4.5")  

---------------------------------------- MPI: use_mpi = ON
-- Found MPI_CXX: /sw/summit/spack-envs/base/opt/linux-rhel8-ppc64le/gcc-9.1.0/spectrum-mpi-10.4.0.3-20210112-6jbupg3thjwhsabgevk6xmwhd2bbyxdc/lib/libmpiprofilesupport.so (found version "3.1") 
-- Found MPI: TRUE (found version "3.1") found components: CXX 
-- Found MPI: /sw/summit/spack-envs/base/opt/linux-rhel8-ppc64le/gcc-9.1.0/spectrum-mpi-10.4.0.3-20210112-6jbupg3thjwhsabgevk6xmwhd2bbyxdc/lib/libmpiprofilesupport.so;/sw/summit/spack-envs/base/opt/linux-rhel8-ppc64le/gcc-9.1.0/spectrum-mpi-10.4.0.3-20210112-6jbupg3thjwhsabgevk6xmwhd2bbyxdc/lib/libmpi_ibm.so

---------------------------------------- BLAS++
-- Using CMAKE_INSTALL_PREFIX = /opt/slate

-- Looking for CUDA (gpu_backend = auto)
-- Looking for C++ include pthread.h
-- Looking for C++ include pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Building CUDA support

-- No HIP/ROCm support: gpu_backend = cuda

-- No oneMKL-SYCL device support: gpu_backend = cuda

-- Found OpenMP_CXX: -fopenmp (found version "4.5") 
-- Found OpenMP_Fortran: -fopenmp (found version "4.5") 
-- blaspp_id = 6b411ed

-- Looking for BLAS libraries and options (blas = auto)
...
IBM ESSL int (lp64), multi-threaded, with OpenMP
   libs:  -lesslsmp
   -DBLAS_FORTRAN_ADD_                              yes
   Found BLAS library: -lesslsmp

-- Checking BLAS library version
   ESSL_VERSION=6.3.0.0
```

Reply:
With this PR, CMake finds libessl.so:
```
[mgates31@login2.summit build-summit]$ git idx
5a481141 (HEAD -> sycl-build-blas-threaded-fix, asim/sycl-build-blas-threaded-fix) Move SYCL search before BLAS options so blas_threaded can be set appropriately.
[mgates31@login2.summit build-summit]$ cmake ..
-- The CXX compiler identification is GNU 9.1.0
-- The Fortran compiler identification is GNU 9.1.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /sw/summit/gcc/9.1.0-alpha+20190716/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting Fortran compiler ABI info
-- Detecting Fortran compiler ABI info - done
-- Check for working Fortran compiler: /sw/summit/gcc/9.1.0-alpha+20190716/bin/gfortran - skipped
-- Setting CMAKE_INSTALL_PREFIX = /opt/slate
---------------------------------------- C API: c_api = OFF
-- slate_id = 5a481141

---------------------------------------- OpenMP: use_openmp = ON
-- Found OpenMP_CXX: -fopenmp (found version "4.5") 
-- Found OpenMP_Fortran: -fopenmp (found version "4.5") 
-- Found OpenMP: TRUE (found version "4.5")  

---------------------------------------- MPI: use_mpi = ON
-- Found MPI_CXX: /sw/summit/spack-envs/base/opt/linux-rhel8-ppc64le/gcc-9.1.0/spectrum-mpi-10.4.0.3-20210112-6jbupg3thjwhsabgevk6xmwhd2bbyxdc/lib/libmpiprofilesupport.so (found version "3.1") 
-- Found MPI: TRUE (found version "3.1") found components: CXX 
-- Found MPI: /sw/summit/spack-envs/base/opt/linux-rhel8-ppc64le/gcc-9.1.0/spectrum-mpi-10.4.0.3-20210112-6jbupg3thjwhsabgevk6xmwhd2bbyxdc/lib/libmpiprofilesupport.so;/sw/summit/spack-envs/base/opt/linux-rhel8-ppc64le/gcc-9.1.0/spectrum-mpi-10.4.0.3-20210112-6jbupg3thjwhsabgevk6xmwhd2bbyxdc/lib/libmpi_ibm.so

---------------------------------------- oneMKL-SYCL: gpu_backend = auto
-- Looking for oneMKL-SYCL
-- Performing Test FSYCL_SUPPORT
-- Performing Test FSYCL_SUPPORT - Failed
-- No oneMKL-SYCL device support: oneMKL or SYCL compiler not found

---------------------------------------- BLAS++
-- Using CMAKE_INSTALL_PREFIX = /opt/slate

-- Looking for CUDA (gpu_backend = auto)
-- Looking for C++ include pthread.h
-- Looking for C++ include pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE  
-- Building CUDA support

-- No HIP/ROCm support: gpu_backend = cuda

-- No oneMKL-SYCL device support: gpu_backend = cuda

-- Found OpenMP_CXX: -fopenmp (found version "4.5") 
-- Found OpenMP_Fortran: -fopenmp (found version "4.5") 
-- blaspp_id = 6b411ed

-- Looking for BLAS libraries and options (blas = auto)
default (no library)
   libs:   
   -DBLAS_FORTRAN_ADD_                              no (didn't link: routine not found)
   -DBLAS_FORTRAN_LOWER                             no (didn't link: routine not found)
   -DBLAS_FORTRAN_UPPER                             no (didn't link: routine not found)

Intel MKL lp64,  sequential, gfortran
   libs:  -lmkl_gf_lp64  -lmkl_sequential -lmkl_core
   -DBLAS_FORTRAN_ADD_                              no (library not found)

Intel MKL ilp64, sequential, gfortran
   libs:  -lmkl_gf_ilp64 -lmkl_sequential -lmkl_core
   -DBLAS_FORTRAN_ADD_                              no (library not found)

Intel MKL lp64,  sequential, ifort
   libs:  -lmkl_intel_lp64  -lmkl_sequential -lmkl_core
   -DBLAS_FORTRAN_ADD_                              no (library not found)

Intel MKL ilp64, sequential, ifort
   libs:  -lmkl_intel_ilp64 -lmkl_sequential -lmkl_core
   -DBLAS_FORTRAN_ADD_                              no (library not found)

IBM ESSL int (lp64), sequential
   libs:  -lessl
   -DBLAS_FORTRAN_ADD_                              yes
   Found BLAS library: -lessl

-- Checking BLAS library version
   ESSL_VERSION=6.3.0.0

-- Checking BLAS complex return type
   BLAS (zdotc) returns complex (GNU gfortran convention)
-- Checking BLAS float return type
   BLAS (sdot) returns float as float (standard)
-- Checking for CBLAS library 
   Found CBLAS library 

-- Looking for LAPACK libraries and options (lapack = auto)
   In BLAS library                                  yes
   Found LAPACK library in BLAS library

-- BLAS++ done

---------------------------------------- LAPACK++
-- Using CMAKE_INSTALL_PREFIX = /opt/slate

-- Looking for CUDA (gpu_backend = auto)
-- Building CUDA support

-- No HIP/ROCm support: gpu_backend = cuda

-- No oneMKL-SYCL device support: gpu_backend = cuda

-- lapackpp_id = d7f699a
-- Check for BLAS++
   BLAS++ already included

-- Looking for LAPACK libraries and options
   In BLAS library                                  no (didn't link: routine not found)
   -llapack                                         yes
   Found LAPACK library: -llapack
-- Checking LAPACK version
   LAPACK version 3.09.01 (30901)
-- Checking for XBLAS
   XBLAS not found.
-- Checking for LAPACKE library 
-- Checking for LAPACKE library -llapacke
   Found LAPACKE library: -llapacke
-- LAPACK++ done

---------------------------------------- CUDA or HIP/ROCm: gpu_backend = auto
-- Looking for CUDA
-- Looking for a CUDA compiler
-- Looking for a CUDA compiler - /sw/summit/cuda/11.0.3/bin/nvcc
-- The CUDA compiler identification is NVIDIA 11.0.221
-- Detecting CUDA compiler ABI info
-- Detecting CUDA compiler ABI info - done
-- Check for working CUDA compiler: /sw/summit/cuda/11.0.3/bin/nvcc - skipped
-- Detecting CUDA compile features
-- Detecting CUDA compile features - done
-- Building CUDA support in SLATE
-- Not building HIP/ROCm support in SLATE

---------------------------------------- ScaLAPACK
-- Using ScaLAPACK library: scalapack

---------------------------------------- TestSweeper
-- Checking for TestSweeper library
-- Using CMAKE_INSTALL_PREFIX = /opt/slate
-- testsweeper_id = db03064
-- TestSweeper done

-- Configuring done
-- Generating done
-- Build files have been written to: /ccs/home/mgates31/repos/slate/build-summit
```

Reply:
It also builds cleanly:
```
[mgates31@login2.summit build-summit]$ make
[  0%] Building CXX object testsweeper/CMakeFiles/testsweeper.dir/testsweeper.cc.o
[  0%] Building CXX object testsweeper/CMakeFiles/testsweeper.dir/version.cc.o
[  0%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/asum.cc.o
[  0%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/batch_her2k.cc.o
[  0%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/batch_gemm.cc.o
[  0%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/axpy.cc.o
[  0%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/batch_hemm.cc.o
[  0%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/batch_trmm.cc.o
[  0%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/batch_symm.cc.o
[  0%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/batch_herk.cc.o
[  0%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/batch_syr2k.cc.o
[  0%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/batch_trsm.cc.o
[  1%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/copy.cc.o
[  1%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/batch_syrk.cc.o
[  1%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/gemm.cc.o
[  1%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/dot.cc.o
[  1%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/gemv.cc.o
[  1%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/ger.cc.o
[  2%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/hemv.cc.o
[  2%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/hemm.cc.o
[  2%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/her.cc.o
[  2%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/her2.cc.o
[  2%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/her2k.cc.o
[  2%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/herk.cc.o
[  2%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/iamax.cc.o
[  2%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/nrm2.cc.o
[  3%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/rot.cc.o
[  3%] Linking CXX shared library libtestsweeper.so
[  3%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/rotg.cc.o
[  3%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/rotm.cc.o
[  3%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/rotmg.cc.o
[  3%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/scal.cc.o
[  3%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/swap.cc.o
[  3%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/symm.cc.o
[  3%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/symv.cc.o
[  4%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/syr.cc.o
[  4%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/syr2.cc.o
[  4%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/syr2k.cc.o
[  4%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/syrk.cc.o
[  4%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/trmm.cc.o
[  4%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/trmv.cc.o
[  4%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/trsm.cc.o
[  4%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/trsv.cc.o
[  5%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/version.cc.o
[  5%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_batch_gemm.cc.o
[  5%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_batch_gemm_group.cc.o
[  5%] Built target testsweeper
[  5%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_batch_hemm.cc.o
[  5%] Building CXX object testsweeper/test/CMakeFiles/testsweeper_tester.dir/test.cc.o
[  5%] Building CXX object testsweeper/test/CMakeFiles/testsweeper_tester.dir/test_sort.cc.o
[  5%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_batch_her2k.cc.o
[  5%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_batch_herk.cc.o
[  5%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_batch_symm.cc.o
[  5%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_batch_syr2k.cc.o
[  6%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_batch_syrk.cc.o
[  6%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_batch_trmm.cc.o
[  6%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_batch_trsm.cc.o
[  6%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_error.cc.o
[  6%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_gemm.cc.o
[  6%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_hemm.cc.o
[  6%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_her2k.cc.o
[  6%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_herk.cc.o
[  7%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_queue.cc.o
[  7%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_symm.cc.o
[  7%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_syr2k.cc.o
[  8%] Linking CXX executable testsweeper_tester
[  8%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_syrk.cc.o
[  8%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_axpy.cc.o
[  8%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_dot.cc.o
[  8%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_nrm2.cc.o
[  8%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_scal.cc.o
[  9%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_swap.cc.o
[  9%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_copy.cc.o
[  9%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_trsm.cc.o
[  9%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_trmm.cc.o
[  9%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/device_utils.cc.o
[  9%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/cublas_wrappers.cc.o
[  9%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/rocblas_wrappers.cc.o
[  9%] Building CXX object blaspp/CMakeFiles/blaspp.dir/src/onemkl_wrappers.cc.o
[  9%] Built target testsweeper_tester
[ 10%] Linking CXX shared library libblaspp.so
[ 10%] Built target blaspp
[ 10%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/bbcsd.cc.o
[ 10%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/bdsdc.cc.o
[ 10%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/bdsvdx.cc.o
[ 10%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/bdsqr.cc.o
[ 10%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/disna.cc.o
[ 11%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gbrfs.cc.o
[ 11%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gbbrd.cc.o
[ 11%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gbequb.cc.o
[ 11%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gbcon.cc.o
[ 11%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gbequ.cc.o
[ 11%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gbrfsx.cc.o
[ 11%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gbsvx.cc.o
[ 11%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gbsv.cc.o
[ 11%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gbtrs.cc.o
[ 11%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gbtrf.cc.o
[ 12%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gebak.cc.o
[ 12%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gebal.cc.o
[ 12%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gebrd.cc.o
[ 12%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gecon.cc.o
[ 12%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/geequ.cc.o
[ 12%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/geequb.cc.o
[ 12%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gees.cc.o
[ 12%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/geesx.cc.o
[ 13%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/geev.cc.o
[ 13%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gehrd.cc.o
[ 13%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gelq.cc.o
[ 13%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gelq2.cc.o
[ 13%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gelqf.cc.o
[ 13%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gels.cc.o
[ 13%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gelsd.cc.o
[ 13%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gelss.cc.o
[ 14%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gelsy.cc.o
[ 14%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gemlq.cc.o
[ 14%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gemqr.cc.o
[ 14%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/geql2.cc.o
[ 14%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/geqlf.cc.o
[ 14%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/geqp3.cc.o
[ 14%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/geqr.cc.o
[ 14%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/geqr2.cc.o
[ 15%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/geqrf.cc.o
[ 15%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/geqrfp.cc.o
[ 15%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/geqrt.cc.o
[ 15%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/geqrt2.cc.o
[ 15%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/geqrt3.cc.o
[ 15%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gerfs.cc.o
[ 15%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gerfsx.cc.o
[ 15%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gerq2.cc.o
[ 16%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gerqf.cc.o
[ 16%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gesdd.cc.o
[ 16%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gesv.cc.o
[ 16%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gesvd.cc.o
[ 16%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gesvdx.cc.o
[ 16%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gesvx.cc.o
[ 16%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/getf2.cc.o
[ 16%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/getrf.cc.o
[ 17%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/getrf2.cc.o
[ 17%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/getri.cc.o
[ 17%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/getrs.cc.o
[ 17%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/getsls.cc.o
[ 17%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ggbak.cc.o
[ 17%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ggbal.cc.o
[ 17%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gges.cc.o
[ 17%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gges3.cc.o
[ 18%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ggesx.cc.o
[ 18%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ggev.cc.o
[ 18%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ggev3.cc.o
[ 18%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ggglm.cc.o
[ 18%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gghrd.cc.o
[ 18%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gglse.cc.o
[ 18%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ggqrf.cc.o
[ 18%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ggrqf.cc.o
[ 19%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ggsvd3.cc.o
[ 19%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ggsvp3.cc.o
[ 19%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gtcon.cc.o
[ 19%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gtrfs.cc.o
[ 19%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gtsv.cc.o
[ 19%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gtsvx.cc.o
[ 19%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gttrf.cc.o
[ 19%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/gttrs.cc.o
[ 20%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hbev_2stage.cc.o
[ 20%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hbev.cc.o
[ 20%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hbevd_2stage.cc.o
[ 20%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hbevd.cc.o
[ 20%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hbevx_2stage.cc.o
[ 20%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hbevx.cc.o
[ 20%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hbgst.cc.o
[ 20%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hbgv.cc.o
[ 21%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hbgvd.cc.o
[ 21%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hbgvx.cc.o
[ 21%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hbtrd.cc.o
[ 21%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hecon_rk.cc.o
[ 21%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hecon.cc.o
[ 21%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/heequb.cc.o
[ 21%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/heev_2stage.cc.o
[ 21%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/heev.cc.o
[ 21%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/heevd_2stage.cc.o
[ 22%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/heevd.cc.o
[ 22%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/heevr_2stage.cc.o
[ 22%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/heevr.cc.o
[ 22%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/heevx_2stage.cc.o
[ 22%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/heevx.cc.o
[ 22%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hegst.cc.o
[ 22%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hegv_2stage.cc.o
[ 22%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hegv.cc.o
[ 23%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hegvd.cc.o
[ 23%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hegvx.cc.o
[ 23%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/herfs.cc.o
[ 23%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/herfsx.cc.o
[ 23%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hesv_aa.cc.o
[ 23%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hesv_rk.cc.o
[ 23%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hesv_rook.cc.o
[ 23%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hesv.cc.o
[ 24%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hesvx.cc.o
[ 24%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/heswapr.cc.o
[ 24%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hetrd_2stage.cc.o
[ 24%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hetrd.cc.o
[ 24%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hetrf_aa.cc.o
[ 24%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hetrf_rk.cc.o
[ 24%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hetrf_rook.cc.o
[ 24%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hetrf.cc.o
[ 25%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hetri_rk.cc.o
[ 25%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hetri.cc.o
[ 25%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hetri2.cc.o
[ 25%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hetrs_aa.cc.o
[ 25%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hetrs_rk.cc.o
[ 25%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hetrs_rook.cc.o
[ 25%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hetrs.cc.o
[ 25%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hetrs2.cc.o
[ 26%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hfrk.cc.o
[ 26%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hgeqz.cc.o
[ 26%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hpcon.cc.o
[ 26%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hpev.cc.o
[ 26%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hpevd.cc.o
[ 26%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hpevx.cc.o
[ 26%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hpgst.cc.o
[ 26%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hpgv.cc.o
[ 27%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hpgvd.cc.o
[ 27%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hpgvx.cc.o
[ 27%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hprfs.cc.o
[ 27%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hpsv.cc.o
[ 27%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hpsvx.cc.o
[ 27%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hptrd.cc.o
[ 27%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hptrf.cc.o
[ 27%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hptri.cc.o
[ 28%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hptrs.cc.o
[ 28%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/hseqr.cc.o
[ 28%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lacgv.cc.o
[ 28%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lacp2.cc.o
[ 28%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lacpy.cc.o
[ 28%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/laed4.cc.o
[ 28%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lag2c.cc.o
[ 28%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lag2d.cc.o
[ 29%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lag2s.cc.o
[ 29%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lag2z.cc.o
[ 29%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lagge.cc.o
[ 29%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/laghe.cc.o
[ 29%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lamch.cc.o
[ 29%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lagsy.cc.o
[ 29%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/langb.cc.o
[ 29%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lange.cc.o
[ 30%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/langt.cc.o
[ 30%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lanhb.cc.o
[ 30%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lanhe.cc.o
[ 30%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lanhp.cc.o
[ 30%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lanhs.cc.o
[ 30%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lanht.cc.o
[ 30%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lansb.cc.o
[ 30%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lansp.cc.o
[ 31%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lanst.cc.o
[ 31%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lansy.cc.o
[ 31%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lantb.cc.o
[ 31%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lantp.cc.o
[ 31%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lantr.cc.o
[ 31%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lapmr.cc.o
[ 31%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lapmt.cc.o
[ 31%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lapy2.cc.o
[ 32%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lapy3.cc.o
[ 32%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/larf.cc.o
[ 32%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/larfb.cc.o
[ 32%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/larfg.cc.o
[ 32%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/larfgp.cc.o
[ 32%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/larft.cc.o
[ 32%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/larfx.cc.o
[ 32%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/larfy.cc.o
[ 33%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/larnv.cc.o
[ 33%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lartg.cc.o
[ 33%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lartgp.cc.o
[ 33%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lartgs.cc.o
[ 33%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lascl.cc.o
[ 33%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lassq.cc.o
[ 33%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/laset.cc.o
[ 33%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/laswp.cc.o
[ 34%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/lauum.cc.o
[ 34%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/opgtr.cc.o
[ 34%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/opmtr.cc.o
[ 34%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/orcsd2by1.cc.o
[ 34%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/orgbr.cc.o
[ 34%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/orghr.cc.o
[ 34%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/orglq.cc.o
[ 34%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/orgql.cc.o
[ 35%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/orgqr.cc.o
[ 35%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/orgrq.cc.o
[ 35%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/orgtr.cc.o
[ 35%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/orhr_col.cc.o
[ 35%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ormbr.cc.o
[ 35%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ormhr.cc.o
[ 35%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ormlq.cc.o
[ 35%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ormql.cc.o
[ 36%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ormqr.cc.o
[ 36%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ormrq.cc.o
[ 36%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ormrz.cc.o
[ 36%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ormtr.cc.o
[ 36%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/pbcon.cc.o
[ 36%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/pbequ.cc.o
[ 36%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/pbrfs.cc.o
[ 36%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/pbstf.cc.o
[ 36%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/pbsvx.cc.o
[ 37%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/pbsv.cc.o
[ 37%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/pbtrf.cc.o
[ 37%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/pbtrs.cc.o
[ 37%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/pftrf.cc.o
[ 37%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/pftri.cc.o
[ 37%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/pftrs.cc.o
[ 37%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/pocon.cc.o
[ 38%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/poequ.cc.o
[ 38%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/poequb.cc.o
[ 38%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/porfs.cc.o
[ 38%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/porfsx.cc.o
[ 38%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/posv.cc.o
[ 38%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/posvx.cc.o
[ 38%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/potf2.cc.o
[ 38%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/potrf.cc.o
[ 39%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/potrf2.cc.o
[ 39%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/potri.cc.o
[ 39%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/potrs.cc.o
[ 39%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ppcon.cc.o
[ 39%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ppequ.cc.o
[ 39%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/pprfs.cc.o
[ 39%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ppsv.cc.o
[ 39%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ppsvx.cc.o
[ 40%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/pptrf.cc.o
[ 40%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/pptri.cc.o
[ 40%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/pptrs.cc.o
[ 40%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/pstrf.cc.o
[ 40%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ptcon.cc.o
[ 40%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/pteqr.cc.o
[ 40%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ptrfs.cc.o
[ 40%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ptsv.cc.o
[ 41%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ptsvx.cc.o
[ 41%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/pttrf.cc.o
[ 41%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/pttrs.cc.o
[ 41%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sbev_2stage.cc.o
[ 41%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sbev.cc.o
[ 41%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sbevd_2stage.cc.o
[ 41%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sbevd.cc.o
[ 41%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sbevx_2stage.cc.o
[ 41%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sbevx.cc.o
[ 42%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sbgst.cc.o
[ 42%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sbgv.cc.o
[ 42%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sbgvd.cc.o
[ 42%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sbgvx.cc.o
[ 42%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sbtrd.cc.o
[ 42%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sfrk.cc.o
[ 42%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/spcon.cc.o
[ 42%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/spev.cc.o
[ 43%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/spevd.cc.o
[ 43%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/spevx.cc.o
[ 43%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/spgst.cc.o
[ 43%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/spgv.cc.o
[ 43%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/spgvd.cc.o
[ 43%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/spgvx.cc.o
[ 43%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sprfs.cc.o
[ 43%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/spsv.cc.o
[ 44%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/spsvx.cc.o
[ 44%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sptrd.cc.o
[ 44%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sptrf.cc.o
[ 44%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sptri.cc.o
[ 44%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sptrs.cc.o
[ 44%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/stedc.cc.o
[ 44%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/stegr.cc.o
[ 44%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/stein.cc.o
[ 45%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/stemr.cc.o
[ 45%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/steqr.cc.o
[ 45%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sterf.cc.o
[ 45%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/stev.cc.o
[ 45%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/stevd.cc.o
[ 45%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/stevr.cc.o
[ 45%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/stevx.cc.o
[ 45%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sturm.cc.o
[ 46%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sycon_rk.cc.o
[ 46%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sycon.cc.o
[ 46%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/syequb.cc.o
[ 46%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/syev_2stage.cc.o
[ 46%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/syev.cc.o
[ 46%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/syevd_2stage.cc.o
[ 46%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/syevd.cc.o
[ 46%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/syevr_2stage.cc.o
[ 47%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/syevr.cc.o
[ 47%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/syevx_2stage.cc.o
[ 47%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/syevx.cc.o
[ 47%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sygst.cc.o
[ 47%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sygv_2stage.cc.o
[ 47%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sygv.cc.o
[ 47%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sygvd.cc.o
[ 47%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sygvx.cc.o
[ 48%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/symv.cc.o
[ 48%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/syr.cc.o
[ 48%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/syrfs.cc.o
[ 48%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/syrfsx.cc.o
[ 48%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sysv_aa.cc.o
[ 48%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sysv_rk.cc.o
[ 48%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sysv_rook.cc.o
[ 48%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sysv.cc.o
[ 49%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sysvx.cc.o
[ 49%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/syswapr.cc.o
[ 49%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sytrd_2stage.cc.o
[ 49%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sytrd.cc.o
[ 49%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sytrf_aa.cc.o
[ 49%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sytrf_rk.cc.o
[ 49%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sytrf_rook.cc.o
[ 49%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sytrf.cc.o
[ 50%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sytri_rk.cc.o
[ 50%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sytri.cc.o
[ 50%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sytri2.cc.o
[ 50%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sytrs_aa.cc.o
[ 50%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sytrs_rk.cc.o
[ 50%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sytrs_rook.cc.o
[ 50%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sytrs.cc.o
[ 50%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/sytrs2.cc.o
[ 51%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tbcon.cc.o
[ 51%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tbrfs.cc.o
[ 51%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tbtrs.cc.o
[ 51%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tfsm.cc.o
[ 51%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tftri.cc.o
[ 51%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tfttp.cc.o
[ 51%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tfttr.cc.o
[ 51%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tgexc.cc.o
[ 52%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tgsen.cc.o
[ 52%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tgsja.cc.o
[ 52%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tgsyl.cc.o
[ 52%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tpcon.cc.o
[ 52%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tplqt.cc.o
[ 52%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tplqt2.cc.o
[ 52%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tpmlqt.cc.o
[ 52%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tpmqrt.cc.o
[ 53%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tpqrt.cc.o
[ 53%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tpqrt2.cc.o
[ 53%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tprfb.cc.o
[ 53%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tprfs.cc.o
[ 53%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tptri.cc.o
[ 53%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tptrs.cc.o
[ 53%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tpttf.cc.o
[ 53%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tpttr.cc.o
[ 54%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/trcon.cc.o
[ 54%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/trevc.cc.o
[ 54%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/trevc3.cc.o
[ 54%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/trexc.cc.o
[ 54%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/trrfs.cc.o
[ 54%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/trsen.cc.o
[ 54%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/trtri.cc.o
[ 54%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/trtrs.cc.o
[ 55%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/trttf.cc.o
[ 55%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/trttp.cc.o
[ 55%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/tzrzf.cc.o
[ 55%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ungbr.cc.o
[ 55%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/unghr.cc.o
[ 55%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/unglq.cc.o
[ 55%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ungql.cc.o
[ 55%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ungqr.cc.o
[ 56%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ungrq.cc.o
[ 56%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/ungtr.cc.o
[ 56%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/unhr_col.cc.o
[ 56%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/unmbr.cc.o
[ 56%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/unmhr.cc.o
[ 56%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/unmlq.cc.o
[ 56%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/unmql.cc.o
[ 56%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/unmqr.cc.o
[ 57%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/unmrq.cc.o
[ 57%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/unmrz.cc.o
[ 57%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/unmtr.cc.o
[ 57%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/upgtr.cc.o
[ 57%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/upmtr.cc.o
[ 57%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/version.cc.o
[ 57%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/cuda/cuda_common.cc.o
[ 57%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/cuda/cuda_geqrf.cc.o
[ 58%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/cuda/cuda_getrf.cc.o
[ 58%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/cuda/cuda_potrf.cc.o
[ 58%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/rocm/rocm_geqrf.cc.o
[ 58%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/rocm/rocm_getrf.cc.o
[ 58%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/rocm/rocm_potrf.cc.o
[ 58%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/onemkl/onemkl_geqrf.cc.o
[ 58%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/onemkl/onemkl_getrf.cc.o
[ 58%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/onemkl/onemkl_potrf.cc.o
[ 59%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/stub/stub_geqrf.cc.o
[ 59%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/stub/stub_getrf.cc.o
[ 59%] Building CXX object lapackpp/CMakeFiles/lapackpp.dir/src/stub/stub_potrf.cc.o
[ 59%] Linking CXX shared library liblapackpp.so
[ 59%] Built target lapackpp
Scanning dependencies of target slate
[ 59%] Building CXX object CMakeFiles/slate.dir/src/add.cc.o
[ 59%] Building CXX object CMakeFiles/slate.dir/src/auxiliary/Debug.cc.o
[ 60%] Building CXX object CMakeFiles/slate.dir/src/colNorms.cc.o
[ 60%] Building CXX object CMakeFiles/slate.dir/src/auxiliary/Trace.cc.o
[ 60%] Building CXX object CMakeFiles/slate.dir/src/bdsqr.cc.o
[ 60%] Building CXX object CMakeFiles/slate.dir/src/cholqr.cc.o
[ 60%] Building CXX object CMakeFiles/slate.dir/src/core/Memory.cc.o
[ 60%] Building CXX object CMakeFiles/slate.dir/src/copy.cc.o
[ 60%] Building CXX object CMakeFiles/slate.dir/src/core/types.cc.o
[ 60%] Building Fortran object CMakeFiles/slate.dir/src/csteqr2.f.o
[ 60%] Building CXX object CMakeFiles/slate.dir/src/device/dev_gescale_row_col.cc.o
[ 60%] Building Fortran object CMakeFiles/slate.dir/src/dsteqr2.f.o
[ 61%] Building CXX object CMakeFiles/slate.dir/src/gbsv.cc.o
[ 61%] Building CXX object CMakeFiles/slate.dir/src/gbmm.cc.o
[ 61%] Building CXX object CMakeFiles/slate.dir/src/gbtrs.cc.o
[ 61%] Building CXX object CMakeFiles/slate.dir/src/gbtrf.cc.o
[ 61%] Building CXX object CMakeFiles/slate.dir/src/ge2tb.cc.o
[ 61%] Building CXX object CMakeFiles/slate.dir/src/gecondest.cc.o
[ 61%] Building CXX object CMakeFiles/slate.dir/src/gelqf.cc.o
[ 61%] Building CXX object CMakeFiles/slate.dir/src/gels.cc.o
[ 61%] Building CXX object CMakeFiles/slate.dir/src/gels_cholqr.cc.o
[ 62%] Building CXX object CMakeFiles/slate.dir/src/gels_qr.cc.o
[ 62%] Building CXX object CMakeFiles/slate.dir/src/gemm.cc.o
[ 62%] Building CXX object CMakeFiles/slate.dir/src/gemmA.cc.o
[ 62%] Building CXX object CMakeFiles/slate.dir/src/gemmC.cc.o
[ 62%] Building CXX object CMakeFiles/slate.dir/src/geqrf.cc.o
[ 62%] Building CXX object CMakeFiles/slate.dir/src/gesv.cc.o
[ 62%] Building CXX object CMakeFiles/slate.dir/src/gesv_mixed.cc.o
[ 62%] Building CXX object CMakeFiles/slate.dir/src/gesv_mixed_gmres.cc.o
[ 63%] Building CXX object CMakeFiles/slate.dir/src/gesv_nopiv.cc.o
[ 63%] Building CXX object CMakeFiles/slate.dir/src/getrf.cc.o
[ 63%] Building CXX object CMakeFiles/slate.dir/src/getrf_nopiv.cc.o
[ 63%] Building CXX object CMakeFiles/slate.dir/src/getrf_tntpiv.cc.o
[ 63%] Building CXX object CMakeFiles/slate.dir/src/getri.cc.o
[ 63%] Building CXX object CMakeFiles/slate.dir/src/getriOOP.cc.o
[ 63%] Building CXX object CMakeFiles/slate.dir/src/getrs.cc.o
[ 63%] Building CXX object CMakeFiles/slate.dir/src/getrs_nopiv.cc.o
[ 64%] Building CXX object CMakeFiles/slate.dir/src/hb2st.cc.o
[ 64%] Building CXX object CMakeFiles/slate.dir/src/hbmm.cc.o
[ 64%] Building CXX object CMakeFiles/slate.dir/src/he2hb.cc.o
[ 64%] Building CXX object CMakeFiles/slate.dir/src/heev.cc.o
[ 64%] Building CXX object CMakeFiles/slate.dir/src/hegst.cc.o
[ 64%] Building CXX object CMakeFiles/slate.dir/src/hegv.cc.o
[ 64%] Building CXX object CMakeFiles/slate.dir/src/hemm.cc.o
[ 64%] Building CXX object CMakeFiles/slate.dir/src/hemmA.cc.o
[ 65%] Building CXX object CMakeFiles/slate.dir/src/hemmC.cc.o
[ 65%] Building CXX object CMakeFiles/slate.dir/src/her2k.cc.o
[ 65%] Building CXX object CMakeFiles/slate.dir/src/herk.cc.o
[ 65%] Building CXX object CMakeFiles/slate.dir/src/hesv.cc.o
[ 65%] Building CXX object CMakeFiles/slate.dir/src/hetrf.cc.o
[ 65%] Building CXX object CMakeFiles/slate.dir/src/hetrs.cc.o
[ 65%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_comm.cc.o
[ 65%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_copyhb2st.cc.o
[ 66%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_copytb2bd.cc.o
[ 66%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_gbnorm.cc.o
[ 66%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_geadd.cc.o
[ 66%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_gebr.cc.o
[ 66%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_gecopy.cc.o
[ 66%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_gemm.cc.o
[ 66%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_gemmA.cc.o
[ 66%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_genorm.cc.o
[ 67%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_geqrf.cc.o
[ 67%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_gescale.cc.o
[ 67%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_gescale_row_col.cc.o
[ 67%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_geset.cc.o
[ 67%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_getrf.cc.o
[ 67%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_getrf_nopiv.cc.o
[ 67%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_getrf_tntpiv.cc.o
[ 67%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_hbnorm.cc.o
[ 68%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_he2hb_gemm.cc.o
[ 68%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_he2hb_hemm.cc.o
[ 68%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_he2hb_her2k_offdiag_ranks.cc.o
[ 68%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_he2hb_trmm.cc.o
[ 68%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_hebr.cc.o
[ 68%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_hegst.cc.o
[ 68%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_hemm.cc.o
[ 68%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_hemmA.cc.o
[ 69%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_henorm.cc.o
[ 69%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_her2k.cc.o
[ 69%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_herk.cc.o
[ 69%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_hettmqr.cc.o
[ 69%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_norm1est.cc.o
[ 69%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_potrf.cc.o
[ 69%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_swap.cc.o
[ 69%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_symm.cc.o
[ 70%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_synorm.cc.o
[ 70%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_syr2k.cc.o
[ 70%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_syrk.cc.o
[ 70%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_transpose.cc.o
[ 70%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_trmm.cc.o
[ 70%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_trnorm.cc.o
[ 70%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_trsm.cc.o
[ 70%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_trsmA.cc.o
[ 71%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_trtri.cc.o
[ 71%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_trtrm.cc.o
[ 71%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_ttlqt.cc.o
[ 71%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_ttmlq.cc.o
[ 71%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_ttmqr.cc.o
[ 71%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_ttqrt.cc.o
[ 71%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_tzadd.cc.o
[ 71%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_tzcopy.cc.o
[ 72%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_tzscale.cc.o
[ 72%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_tzset.cc.o
[ 72%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_unmlq.cc.o
[ 72%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_unmqr.cc.o
[ 72%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_unmtr_hb2st.cc.o
[ 72%] Building CXX object CMakeFiles/slate.dir/src/internal/internal_util.cc.o
[ 72%] Building CXX object CMakeFiles/slate.dir/src/norm.cc.o
[ 72%] Building CXX object CMakeFiles/slate.dir/src/pbsv.cc.o
[ 73%] Building CXX object CMakeFiles/slate.dir/src/pbtrf.cc.o
[ 73%] Building CXX object CMakeFiles/slate.dir/src/pbtrs.cc.o
[ 73%] Building CXX object CMakeFiles/slate.dir/src/posv.cc.o
[ 73%] Building CXX object CMakeFiles/slate.dir/src/posv_mixed.cc.o
[ 73%] Building CXX object CMakeFiles/slate.dir/src/posv_mixed_gmres.cc.o
[ 73%] Building CXX object CMakeFiles/slate.dir/src/potrf.cc.o
[ 73%] Building CXX object CMakeFiles/slate.dir/src/potri.cc.o
[ 73%] Building CXX object CMakeFiles/slate.dir/src/potrs.cc.o
[ 74%] Building CXX object CMakeFiles/slate.dir/src/print.cc.o
[ 74%] Building CXX object CMakeFiles/slate.dir/src/redistribute.cc.o
[ 74%] Building CXX object CMakeFiles/slate.dir/src/scale.cc.o
[ 74%] Building CXX object CMakeFiles/slate.dir/src/scale_row_col.cc.o
[ 74%] Building CXX object CMakeFiles/slate.dir/src/set.cc.o
[ 74%] Building Fortran object CMakeFiles/slate.dir/src/ssteqr2.f.o
[ 74%] Building CXX object CMakeFiles/slate.dir/src/stedc.cc.o
[ 74%] Building CXX object CMakeFiles/slate.dir/src/stedc_deflate.cc.o
[ 75%] Building CXX object CMakeFiles/slate.dir/src/stedc_secular.cc.o
[ 75%] Building CXX object CMakeFiles/slate.dir/src/stedc_merge.cc.o
[ 75%] Building CXX object CMakeFiles/slate.dir/src/stedc_solve.cc.o
[ 75%] Building CXX object CMakeFiles/slate.dir/src/stedc_sort.cc.o
[ 75%] Building CXX object CMakeFiles/slate.dir/src/stedc_z_vector.cc.o
[ 75%] Building CXX object CMakeFiles/slate.dir/src/steqr2.cc.o
[ 75%] Building CXX object CMakeFiles/slate.dir/src/sterf.cc.o
[ 75%] Building CXX object CMakeFiles/slate.dir/src/svd.cc.o
[ 76%] Building CXX object CMakeFiles/slate.dir/src/symm.cc.o
[ 76%] Building CXX object CMakeFiles/slate.dir/src/syr2k.cc.o
[ 76%] Building CXX object CMakeFiles/slate.dir/src/syrk.cc.o
[ 76%] Building CXX object CMakeFiles/slate.dir/src/tb2bd.cc.o
[ 76%] Building CXX object CMakeFiles/slate.dir/src/tbsm.cc.o
[ 76%] Building CXX object CMakeFiles/slate.dir/src/tbsmPivots.cc.o
[ 76%] Building CXX object CMakeFiles/slate.dir/src/trcondest.cc.o
[ 76%] Building CXX object CMakeFiles/slate.dir/src/trmm.cc.o
[ 77%] Building CXX object CMakeFiles/slate.dir/src/trsm.cc.o
[ 77%] Building CXX object CMakeFiles/slate.dir/src/trsmA.cc.o
[ 77%] Building CXX object CMakeFiles/slate.dir/src/trsmB.cc.o
[ 77%] Building CXX object CMakeFiles/slate.dir/src/trtri.cc.o
[ 77%] Building CXX object CMakeFiles/slate.dir/src/trtrm.cc.o
[ 77%] Building CXX object CMakeFiles/slate.dir/src/unmbr_ge2tb.cc.o
[ 77%] Building CXX object CMakeFiles/slate.dir/src/unmlq.cc.o
[ 77%] Building CXX object CMakeFiles/slate.dir/src/unmqr.cc.o
[ 78%] Building CXX object CMakeFiles/slate.dir/src/unmtr_hb2st.cc.o
[ 78%] Building CXX object CMakeFiles/slate.dir/src/unmtr_he2hb.cc.o
[ 78%] Building CXX object CMakeFiles/slate.dir/src/version.cc.o
[ 78%] Building CXX object CMakeFiles/slate.dir/src/work/work_trmm.cc.o
[ 78%] Building CXX object CMakeFiles/slate.dir/src/work/work_trsm.cc.o
[ 78%] Building CXX object CMakeFiles/slate.dir/src/work/work_trsmA.cc.o
[ 78%] Building Fortran object CMakeFiles/slate.dir/src/zsteqr2.f.o
[ 78%] Building CUDA object CMakeFiles/slate.dir/src/cuda/device_geadd.cu.o
[ 79%] Building CUDA object CMakeFiles/slate.dir/src/cuda/device_gecopy.cu.o
[ 79%] Building CUDA object CMakeFiles/slate.dir/src/cuda/device_genorm.cu.o
[ 79%] Building CUDA object CMakeFiles/slate.dir/src/cuda/device_gescale.cu.o
[ 79%] Building CUDA object CMakeFiles/slate.dir/src/cuda/device_gescale_row_col.cu.o
[ 79%] Building CUDA object CMakeFiles/slate.dir/src/cuda/device_geset.cu.o
[ 79%] Building CUDA object CMakeFiles/slate.dir/src/cuda/device_henorm.cu.o
[ 79%] Building CUDA object CMakeFiles/slate.dir/src/cuda/device_synorm.cu.o
[ 79%] Building CUDA object CMakeFiles/slate.dir/src/cuda/device_transpose.cu.o
[ 80%] Building CUDA object CMakeFiles/slate.dir/src/cuda/device_trnorm.cu.o
[ 80%] Building CUDA object CMakeFiles/slate.dir/src/cuda/device_tzadd.cu.o
[ 80%] Building CUDA object CMakeFiles/slate.dir/src/cuda/device_tzcopy.cu.o
[ 80%] Building CUDA object CMakeFiles/slate.dir/src/cuda/device_tzscale.cu.o
[ 80%] Building CUDA object CMakeFiles/slate.dir/src/cuda/device_tzset.cu.o
[ 80%] Linking CXX shared library libslate.so
[ 80%] Built target slate
Scanning dependencies of target tester
[ 80%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_gels.cc.o
[ 80%] Building CXX object unit_test/CMakeFiles/test_BandMatrix.dir/test_BandMatrix.cc.o
[ 80%] Building CXX object unit_test/CMakeFiles/test_SymmetricMatrix.dir/test_SymmetricMatrix.cc.o
[ 80%] Building CXX object unit_test/CMakeFiles/test_TriangularMatrix.dir/test_TriangularMatrix.cc.o
[ 80%] Building CXX object unit_test/CMakeFiles/test_OmpSetMaxActiveLevels.dir/test_OmpSetMaxActiveLevels.cc.o
[ 80%] Building CXX object unit_test/CMakeFiles/test_Tile.dir/test_Tile.cc.o
[ 80%] Building CXX object unit_test/CMakeFiles/test_LockGuard.dir/test_LockGuard.cc.o
[ 80%] Building CXX object unit_test/CMakeFiles/test_gecopy.dir/test_gecopy.cc.o
[ 80%] Building CXX object unit_test/CMakeFiles/test_HermitianMatrix.dir/test_HermitianMatrix.cc.o
[ 80%] Building CXX object unit_test/CMakeFiles/test_TrapezoidMatrix.dir/test_TrapezoidMatrix.cc.o
[ 81%] Building CXX object unit_test/CMakeFiles/test_TriangularBandMatrix.dir/test_TriangularBandMatrix.cc.o
[ 81%] Building CXX object unit_test/CMakeFiles/test_Tile_kernels.dir/test_Tile_kernels.cc.o
[ 82%] Building CXX object unit_test/CMakeFiles/test_geadd.dir/test_geadd.cc.o
[ 82%] Building CXX object unit_test/CMakeFiles/test_Memory.dir/test_Memory.cc.o
[ 82%] Building CXX object unit_test/CMakeFiles/test_Matrix.dir/test_Matrix.cc.o
[ 82%] Building CXX object test/CMakeFiles/tester.dir/matrix_generator.cc.o
[ 82%] Building CXX object unit_test/CMakeFiles/test_OmpSetMaxActiveLevels.dir/unit_test.cc.o
[ 82%] Building CXX object unit_test/CMakeFiles/test_LockGuard.dir/unit_test.cc.o
[ 82%] Building CXX object unit_test/CMakeFiles/test_Memory.dir/unit_test.cc.o
[ 82%] Building CXX object unit_test/CMakeFiles/test_gecopy.dir/unit_test.cc.o
[ 82%] Linking CXX executable test_OmpSetMaxActiveLevels
[ 82%] Linking CXX executable test_LockGuard
[ 82%] Building CXX object unit_test/CMakeFiles/test_Tile.dir/unit_test.cc.o
[ 82%] Linking CXX executable test_Memory
[ 82%] Building CXX object unit_test/CMakeFiles/test_TriangularBandMatrix.dir/unit_test.cc.o
[ 82%] Building CXX object unit_test/CMakeFiles/test_Tile_kernels.dir/unit_test.cc.o
[ 82%] Linking CXX executable test_gecopy
[ 82%] Built target test_OmpSetMaxActiveLevels
[ 82%] Building CXX object unit_test/CMakeFiles/test_Matrix.dir/unit_test.cc.o
[ 82%] Built target test_LockGuard
[ 83%] Building CXX object unit_test/CMakeFiles/test_TrapezoidMatrix.dir/unit_test.cc.o
[ 83%] Building CXX object unit_test/CMakeFiles/test_geadd.dir/unit_test.cc.o
[ 83%] Building CXX object unit_test/CMakeFiles/test_SymmetricMatrix.dir/unit_test.cc.o
[ 83%] Linking CXX executable test_Tile
[ 83%] Built target test_Memory
[ 83%] Building CXX object test/CMakeFiles/tester.dir/matrix_params.cc.o
[ 83%] Building CXX object unit_test/CMakeFiles/test_BandMatrix.dir/unit_test.cc.o
[ 84%] Building CXX object unit_test/CMakeFiles/test_HermitianMatrix.dir/unit_test.cc.o
[ 84%] Linking CXX executable test_TriangularBandMatrix
[ 84%] Building CXX object unit_test/CMakeFiles/test_TriangularMatrix.dir/unit_test.cc.o
[ 84%] Linking CXX executable test_Tile_kernels
[ 84%] Built target test_gecopy
[ 84%] Building CXX object test/CMakeFiles/tester.dir/random.cc.o
[ 84%] Building CXX object test/CMakeFiles/tester.dir/test.cc.o
[ 85%] Linking CXX executable test_SymmetricMatrix
[ 85%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_gemm.cc.o
[ 85%] Linking CXX executable test_geadd
[ 85%] Linking CXX executable test_HermitianMatrix
[ 85%] Building CXX object unit_test/CMakeFiles/test_gescale.dir/test_gescale.cc.o
[ 85%] Linking CXX executable test_TrapezoidMatrix
[ 85%] Built target test_Tile
[ 85%] Building CXX object unit_test/CMakeFiles/test_gescale.dir/unit_test.cc.o
[ 85%] Built target test_TriangularBandMatrix
[ 85%] Linking CXX executable test_BandMatrix
[ 85%] Building CXX object test/CMakeFiles/tester.dir/test_add.cc.o
[ 85%] Linking CXX executable test_TriangularMatrix
[ 85%] Building CXX object unit_test/CMakeFiles/test_geset.dir/test_geset.cc.o
[ 85%] Building CXX object unit_test/CMakeFiles/test_geset.dir/unit_test.cc.o
[ 85%] Built target test_Tile_kernels
[ 86%] Building CXX object test/CMakeFiles/tester.dir/test_bdsqr.cc.o
[ 86%] Linking CXX executable test_Matrix
[ 86%] Built target test_SymmetricMatrix
[ 86%] Building CXX object test/CMakeFiles/tester.dir/test_copy.cc.o
[ 86%] Building CXX object unit_test/CMakeFiles/test_internal_blas.dir/test_internal_blas.cc.o
[ 86%] Built target test_geadd
[ 86%] Building CXX object unit_test/CMakeFiles/test_internal_blas.dir/unit_test.cc.o
[ 86%] Built target test_HermitianMatrix
[ 86%] Building CXX object test/CMakeFiles/tester.dir/test_gbmm.cc.o
[ 86%] Built target test_TrapezoidMatrix
[ 87%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_gesv.cc.o
[ 87%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_gesv_mixed.cc.o
[ 87%] Built target test_BandMatrix
[ 87%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_getrf.cc.o
[ 87%] Built target test_TriangularMatrix
[ 87%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_getri.cc.o
[ 87%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_getrs.cc.o
[ 87%] Built target test_Matrix
[ 87%] Building CXX object test/CMakeFiles/tester.dir/test_gbnorm.cc.o
[ 88%] Linking CXX executable test_gescale
[ 88%] Building CXX object unit_test/CMakeFiles/test_lq.dir/test_lq.cc.o
[ 89%] Building CXX object unit_test/CMakeFiles/test_lq.dir/unit_test.cc.o
[ 89%] Linking CXX executable test_geset
[ 89%] Built target test_gescale
[ 89%] Building CXX object test/CMakeFiles/tester.dir/test_gbsv.cc.o
[ 89%] Building CXX object test/CMakeFiles/tester.dir/test_ge2tb.cc.o
[ 89%] Building CXX object test/CMakeFiles/tester.dir/test_gecondest.cc.o
[ 89%] Built target test_geset
[ 89%] Building CXX object test/CMakeFiles/tester.dir/test_gelqf.cc.o
[ 89%] Building CXX object unit_test/CMakeFiles/test_norm.dir/test_norm.cc.o
[ 89%] Linking CXX executable test_internal_blas
[ 89%] Building CXX object unit_test/CMakeFiles/test_norm.dir/unit_test.cc.o
[ 89%] Building CXX object unit_test/CMakeFiles/test_qr.dir/test_qr.cc.o
[ 89%] Building CXX object unit_test/CMakeFiles/test_qr.dir/unit_test.cc.o
[ 90%] Building CXX object unit_test/CMakeFiles/test_util.dir/test_util.cc.o
[ 90%] Built target test_internal_blas
[ 90%] Building CXX object unit_test/CMakeFiles/test_util.dir/unit_test.cc.o
[ 90%] Building CXX object test/CMakeFiles/tester.dir/test_gemm.cc.o
[ 91%] Building CXX object test/CMakeFiles/tester.dir/test_gels.cc.o
[ 91%] Building CXX object test/CMakeFiles/tester.dir/test_genorm.cc.o
[ 91%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_hemm.cc.o
[ 91%] Linking CXX executable test_norm
[ 91%] Building CXX object test/CMakeFiles/tester.dir/test_geqrf.cc.o
[ 91%] Building CXX object test/CMakeFiles/tester.dir/test_gesv.cc.o
[ 91%] Building CXX object test/CMakeFiles/tester.dir/test_getri.cc.o
[ 91%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_her2k.cc.o
[ 91%] Linking CXX executable test_util
[ 91%] Built target test_norm
[ 91%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_herk.cc.o
[ 91%] Built target test_util
[ 91%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_lange.cc.o
[ 92%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_lanhe.cc.o
[ 92%] Linking CXX executable test_lq
[ 92%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_lansy.cc.o
[ 92%] Building CXX object test/CMakeFiles/tester.dir/test_hb2st.cc.o
[ 92%] Building CXX object test/CMakeFiles/tester.dir/test_hbmm.cc.o
[ 92%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_lantr.cc.o
[ 92%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_posv.cc.o
[ 92%] Built target test_lq
[ 92%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_potrf.cc.o
[ 92%] Linking CXX executable test_qr
[ 93%] Building CXX object test/CMakeFiles/tester.dir/test_hbnorm.cc.o
[ 93%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_potri.cc.o
[ 93%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_symm.cc.o
[ 93%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_syr2k.cc.o
[ 93%] Building CXX object test/CMakeFiles/tester.dir/test_he2hb.cc.o
[ 94%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_syrk.cc.o
[ 94%] Built target test_qr
[ 94%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_trmm.cc.o
[ 94%] Building CXX object CMakeFiles/slate_lapack_api.dir/lapack_api/lapack_trsm.cc.o
[ 94%] Building CXX object test/CMakeFiles/tester.dir/test_heev.cc.o
[ 94%] Building CXX object test/CMakeFiles/tester.dir/test_hegst.cc.o
[ 94%] Building CXX object test/CMakeFiles/tester.dir/test_hegv.cc.o
[ 94%] Building CXX object test/CMakeFiles/tester.dir/test_hemm.cc.o
[ 94%] Building CXX object test/CMakeFiles/tester.dir/test_henorm.cc.o
[ 94%] Building CXX object test/CMakeFiles/tester.dir/test_her2k.cc.o
[ 95%] Building CXX object test/CMakeFiles/tester.dir/test_herk.cc.o
[ 95%] Building CXX object test/CMakeFiles/tester.dir/test_hesv.cc.o
[ 95%] Building CXX object test/CMakeFiles/tester.dir/test_pbsv.cc.o
[ 95%] Building CXX object test/CMakeFiles/tester.dir/test_posv.cc.o
[ 95%] Building CXX object test/CMakeFiles/tester.dir/test_potri.cc.o
[ 95%] Building CXX object test/CMakeFiles/tester.dir/test_scale.cc.o
[ 95%] Building CXX object test/CMakeFiles/tester.dir/test_scale_row_col.cc.o
[ 95%] Linking CXX shared library libslate_lapack_api.so
[ 95%] Building CXX object test/CMakeFiles/tester.dir/test_set.cc.o
[ 96%] Building CXX object test/CMakeFiles/tester.dir/test_stedc.cc.o
[ 96%] Building CXX object test/CMakeFiles/tester.dir/test_stedc_deflate.cc.o
[ 96%] Building CXX object test/CMakeFiles/tester.dir/test_stedc_secular.cc.o
[ 96%] Building CXX object test/CMakeFiles/tester.dir/test_stedc_sort.cc.o
[ 96%] Building CXX object test/CMakeFiles/tester.dir/test_stedc_z_vector.cc.o
[ 96%] Building CXX object test/CMakeFiles/tester.dir/test_steqr2.cc.o
[ 96%] Built target slate_lapack_api
[ 96%] Building CXX object test/CMakeFiles/tester.dir/test_sterf.cc.o
[ 96%] Building CXX object test/CMakeFiles/tester.dir/test_svd.cc.o
[ 97%] Building CXX object test/CMakeFiles/tester.dir/test_symm.cc.o
[ 97%] Building CXX object test/CMakeFiles/tester.dir/test_synorm.cc.o
[ 97%] Building CXX object test/CMakeFiles/tester.dir/test_syr2k.cc.o
[ 97%] Building CXX object test/CMakeFiles/tester.dir/test_syrk.cc.o
[ 97%] Building CXX object test/CMakeFiles/tester.dir/test_tb2bd.cc.o
[ 97%] Building CXX object test/CMakeFiles/tester.dir/test_tbsm.cc.o
[ 97%] Building CXX object test/CMakeFiles/tester.dir/test_trcondest.cc.o
[ 97%] Building CXX object test/CMakeFiles/tester.dir/test_trmm.cc.o
[ 98%] Building CXX object test/CMakeFiles/tester.dir/test_trnorm.cc.o
[ 98%] Building CXX object test/CMakeFiles/tester.dir/test_trsm.cc.o
[ 98%] Building CXX object test/CMakeFiles/tester.dir/test_trtri.cc.o
[ 98%] Building CXX object test/CMakeFiles/tester.dir/test_unmqr.cc.o
[ 98%] Building CXX object test/CMakeFiles/tester.dir/test_unmtr_hb2st.cc.o
[ 98%] Building CXX object test/CMakeFiles/tester.dir/test_unmtr_he2hb.cc.o
[ 98%] Building Fortran object test/CMakeFiles/tester.dir/pclange.f.o
[ 98%] Building Fortran object test/CMakeFiles/tester.dir/pclansy.f.o
[ 99%] Building Fortran object test/CMakeFiles/tester.dir/pclantr.f.o
[ 99%] Building Fortran object test/CMakeFiles/tester.dir/pdlange.f.o
[ 99%] Building Fortran object test/CMakeFiles/tester.dir/pdlansy.f.o
[ 99%] Building Fortran object test/CMakeFiles/tester.dir/pdlantr.f.o
[ 99%] Building Fortran object test/CMakeFiles/tester.dir/pslange.f.o
[ 99%] Building Fortran object test/CMakeFiles/tester.dir/pslansy.f.o
[ 99%] Building Fortran object test/CMakeFiles/tester.dir/pslantr.f.o
[ 99%] Building Fortran object test/CMakeFiles/tester.dir/pzlange.f.o
[100%] Building Fortran object test/CMakeFiles/tester.dir/pzlansy.f.o
[100%] Building Fortran object test/CMakeFiles/tester.dir/pzlantr.f.o
[100%] Linking CXX executable tester
[100%] Built target tester
```